docker-traffic-gen/run_traffic_gen.sh

```diff
@@ -1,23 +1,36 @@
-#!/bin/sh
+#!/bin/bash
+#
+# Randomly pick an url to fetch and then a semi random pause up to 5s.
+# Use http url so we capture both clear and usually encrypted version
+# after a redirect.
 
 set -e
 
+URLS=(
+  http://www.google.com
+  http://www.youtube.com
+  http://www.twitter.com
+  http://www.wikipedia.org
+  http://www.amazon.com
+  http://www.instagram.com
+  http://www.linkedin.com
+  http://www.reddit.com
+  http://www.whatsapp.com
+  http://openai.com
+  http://www.yahoo.com
+  http://www.bing.com
+  http://www.live.com
+  http://www.microsoft.com
+)
+
+SLEEPS=(0 0.1 0.5 1 1.5 2 3 5)
+
 # Curl some Alexa top 100 sites, wash, repeat
 while :
 do
-	curl https://www.google.com
-	curl https://www.youtube.com
-	curl https://www.twitter.com
-	curl https://www.wikipedia.org
-	curl https://www.amazon.com
-	curl https://www.instagram.com
-	curl https://www.linkedin.com
-	curl https://www.reddit.com
-	curl https://www.whatsapp.com
-	curl https://openai.com
-	curl https://www.yahoo.com
-	curl https://www.bing.com
-	curl https://www.live.com
-	curl https://www.microsoft.com
-	sleep 10
-done
\ No newline at end of file
+    u=${URLS[$RANDOM % ${#URLS[@]}]}
+    s=${SLEEPS[$RANDOM % ${#SLEEPS[@]}]}
+    #echo $s - $u
+    curl -L --output /dev/null $u
+    sleep $s
```

Comment:
Didn't see any notes on testing in the description.  Did you check it locally or in Fargate?

Reply:
I only tested with ./run_traffic_get.sh and inside docker, not fargate.

Reply:
`./manage_arkime.py deploy-demo-traffic` worked, not sure how to see if script is actually running or not

Reply:
I think the RANDOM and other features are bash only. But this works fine in my docker, so it should be the same image right?

Reply:
It should. You can actually shell into the Fargate task and poke around using the AWS Console.  Quite helpful for debugging "Works on my Box" issues.